package.json

```diff
@@ -1,12 +1,41 @@
 {
+  "name": "alfred-emoj",
+  "version": "0.1.3",
+  "description": "Alfred 3 workflow to find relevant emoji from text",
+  "license": "MIT",
+  "repository": "sindresorhus/alfred-emoj",
+  "author": {
+    "name": "Sindre Sorhus",
+    "email": "sindresorhus@gmail.com",
+    "url": "sindresorhus.com"
+  },
+  "engines": {
+    "node": ">=4"
+  },
   "scripts": {
-    "test": "xo && ava"
+    "test": "xo && ava",
+    "postinstall": "alfred-link"
   },
+  "files": [
+    "index.js",
+    "icon.png",
+    "info.plist"
+  ],
+  "keywords": [
+    "emoj",
+    "emoji",
```

Comment:
Are these extras or replacements? I think they are extras, just want to be sure :).